### ISSUE TITLE
About distribution of angles and dihedral angels

### ISSUE THREAD

**Issue by ljl123ljl** [pengxingang/Pocket2Mol#20]
Hi, wonderful work!
I wonder if the mols generated follow the same angle distribution and dihedral angel distribution with real mols.

**Comment by pengxingang** [comment]
Yes, Pocket2Mol can generate molecules with similar distributions of bond angle and dihedral angle with the real molecules.
